metacat-common-server/src/main/java/com/netflix/metacat/common/server/events/MetacatEventBus.java

```diff
@@ -36,26 +37,23 @@
  * @since 0.x
  */
 @Slf4j
-public class MetacatEventBus {
+public class MetacatEventBus extends SimpleApplicationEventMulticaster {
 
-    private final ApplicationEventPublisher eventPublisher;
-    private final ApplicationEventMulticaster eventMulticaster;
+    private final ApplicationEventMulticaster asyncEventMulticaster;
     private final Counter eventAsyncCounter;
     private final Counter eventSyncCounter;
     /**
      * Constructor.
      *
-     * @param eventPublisher   The synchronous event publisher to use
-     * @param eventMulticaster The asynchronous event multicaster to use
+     * @param asyncEventMulticaster The asynchronous event multicaster to use
      * @param registry         The registry to spectator
      */
     public MetacatEventBus(
-            @Nonnull @NonNull final ApplicationEventPublisher eventPublisher,
-            @Nonnull @NonNull final ApplicationEventMulticaster eventMulticaster,
+            @Nonnull @NonNull final ApplicationEventMulticaster asyncEventMulticaster,
```

Comment:
If you get rid of the synchronous one how are you sending the synchronous events?

Reply:
MetacatEventBus is the default event publisher that processes events synchronously. I do not want to create another event publisher within it.